import { AuthenticationException } from "./authenticationException";

export class UserNotFoundException extends AuthenticationException {
    constructor(login: string) {
        super(login, "Usu√°rio n√£o encontrado!");

        // üëáÔ∏è because we are extending a built-in class
        Object.setPrototypeOf(this, UserNotFoundException.prototype);
    }
}